src/PIL/Image.py

```diff
@@ -633,19 +633,34 @@ def _repr_pretty_(self, p, cycle):
             )
         )
 
-    def _repr_png_(self):
+    def _repr_image(self, image_format):
         """iPython display hook support
 
```

Comment:
```suggestion
        """
```
I feel like the current version implies that iPython uses `_repr_image` directly, which is not the case.

Reply:
```suggestion
        """Helper function for iPython display hook
```
How about this to keep a helpful message without the misleading implication?